database/mysql/mysql_replication.py

```diff
@@ -103,7 +103,10 @@
     master_ssl_cipher:
         description:
             - same as mysql variable
-
+    master_auto_position:
+        descrtiption:
+            - does the host uses GTID based replication or not
+        possible values: 0,1
```

Comment:
you are missing requried: False and default: , also use choices: instead of possible values: